src/test/java/org/folio/tlib/postgres/TenantPgPoolTest.java

```diff
@@ -216,4 +217,22 @@ public void testSSL(TestContext context) throws IOException {
     }));
   }
 
+  @Test
+  public void size() {
+    TenantPgPool pool = TenantPgPool.pool(vertx, "diku");
+    assertThat(pool.size(), is(0));
+  }
+
+  @Test
+  public void close(TestContext context) {
+    TenantPgPool pool = TenantPgPool.pool(vertx, "diku");
+    pool.close(context.asyncAssertSuccess());
+  }
+
+  @Test
+  public void connectHandler(TestContext context) {
+    TenantPgPool pool = TenantPgPool.pool(vertx, "diku");
+    pool.connectHandler(null);
```

Comment:
Can you check that passed connectHandler is run?
```
    pool.connectHandler(conn -> {
      conn.query("CREATE TEMP TABLE connecthandler()").execute()
      .compose(x -> conn.close());
    });
    pool.withConnection(conn -> conn.preparedQuery("SELECT * FROM connecthandler").execute())
    .onComplete(context.asyncAssertSuccess(rowSet -> {
      assertThat(rowSet.size(), is(0));
    }));
```

Reply:
okay.. added (`eventually` instead of `compose` for the close)

Reply:
ok